src/gatling/simulations/uk/gov/hmcts/reform/sscs/performance/trackyourappeal/SSCSMYA.scala

```diff
@@ -31,7 +31,7 @@ object SSCSMYA {
   // =======================================================================================
   val login =
   exec(http("SSCSMYA${service}_020_005_LoginPage")
-       .post(idamUrl + "/login?redirect_uri=https%3A%2F%2Fsscs-cor.perftest.platform.hmcts.net%2Fsign-in&client_id=sscs&response_type=code&state=${reference}")
+       .post(idamUrl + "/login?redirect_uri=https%3A%2F%2Fsscs-cor.aat.platform.hmcts.net%2Fsign-in&client_id=sscs&response_type=code&state=${reference}")
```

Comment:
Parameterise this please - declare it as a var in your Environment.scala file and then substitute it